### PR TITLE
Remove the reset of upper layer humidity

### LINKED ISSUE #2448: Remove the reset of the upper layer humidity when cold starting
### What new functionality do you need?

The parameter `nudge_qv` within the FV3 core resets the upper layer humidity to HALOE climatology.  It only resets during a cold start.

GFSv14 stratosphere was too moist.  This option was turned on for GFSv15. GFSv16 raised the model top but frequently used ICs with a lower model top.  The `nudge_qv` parameter remained on for GFSv16.

This option is still on by default in current GFSv17 experiments.  This impacts any forecasts from cold starts, including the prototypes and HR experiments as well as lower resolution DA cycling experiments.  An [examination](https://docs.google.com/presentation/d/1CYPjqs3RCD1uyK2KCXtwUBqiTGm2SLGDkUkwu7PJ5bo/edit?usp=sharing) of operational humidity as well as the spin up in both v16 cycling and v17 free forecast experiments confirms that this parameter can be turned off.

### What are the requirements for the new functionality?

Set `nudge_qv=.false.` by default in GFSv17-based forecasts.

### Acceptance Criteria

Set `nudge_qv=.false.` by default in GFSv17-based forecasts.

### Suggest a solution (optional)

Update `ush/parsing_namelists_FV3.sh`

### PR DESCRIPTION
**Pull request by CatherineThomas-NOAA**
# Description
The parameter "nudge_qv" resets the upper layer humidity to HALOE climatology when cold starting.  This parameter has been set to ".true." but is no longer needed using v16+ ICs and will now be set to ".false." by default.

Resolves: #2448

# Type of change
- Removing feature

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
Cycled test on Hera

# Checklist
- [ ] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Comment by CatherineThomas-NOAA** [comment]
@yangfanglin For awareness

**Review by aerorahul — APPROVED** [review]
looks good. 
Will wait for reviews from GEFS and physics team members to ensure they are not adversely affected.

**Review by yangfanglin — APPROVED** [review]

**Comment by yangfanglin** [comment]
We had discussions about this change at a physics meeting Cathy attended.  It should be applied to both GEFS and GFS applicaitons

**Pull request merged**
